### PR TITLE
daemon stop fix

### LINKED ISSUE #868: daemon does not stop after GUI quits
since a week or so, when I close the wallet it does not always quit the daemon

### PR DESCRIPTION
**Pull request by vikas-cis**
#868

### CODE REVIEW

File: modules/daemon/daemon.js [hunk 1]
```diff
@@ -147,23 +147,20 @@ exports.check = function() {
...
-        } else {
-          log.debug('Daemon stopping gracefully...');
-          resolve();
-        }
-      });
-    } else
-    {
-        log.debug('Daemon not managed by gui.');
+    // Stop deamon either ways
+    rpc.call('stop', null, (error, response) => {
```
[the remark this answers is not in the document]
Comment (vikas-cis) [reply modules/daemon/daemon.js]: what happens if we call this in both case (own daemon or not) ?

[the remark this answers is not in the document]
Comment (kewde) [reply modules/daemon/daemon.js]: The daemon stops but we only want to stop daemons that we started locally.

**Comment by vikas-cis** [comment]
@kewde the daemon stops correctly when it starts via the gui but when it start from somewhere else then some time daemon doesn't stop 

Thanks

**Comment by kewde** [comment]
I'll fix it.

**Pull request closed without merging**